test/Rhetos.LightDMS.IntegrationTest/DownloadS3StorageTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Microsoft.AspNetCore.Mvc.Testing;
+using Rhetos.LightDMS.IntegrationTest.Utilities;
+using Rhetos.LightDMS.TestApp;
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Rhetos.LightDMS.IntegrationTest
+{
+    public class RhetosEntiy
+    {
+        public string ID { get; set; }
+    }

```

Comment:
I don't see this class used, can we remove it?

Reply:
I forget remove it, kkk.